features/editor_url.feature

```diff
@@ -47,14 +47,14 @@ Feature: Editor URL
       | --editor-url | 'phpstorm://open?file={absPath}&line={line}' |
     Then the output should contain:
       """
-        Scenario:                                    # <href=phpstorm://open?file={BASE_PATH}tests/Fixtures/EditorUrl/features/test.feature&line=3>features/test.feature:3</>
+        Scenario:                                    # <href=phpstorm://open?file=%%WORKING_DIR%%features/test.feature&line=3>features/test.feature:3</>
```

Comment:
As a separate PR, I might look at standardising / simplifying the way our FeatureContext handles the various placeholders for directories and filesystem paths - they are not quite consistent with each other, and I think some might be redundant.

But for the purpose of this PR, all paths in features will now be relative to the (previously defined) %%WORKING_DIR%% placeholder.